CFC_DataCollector/recommender/trip.py

```diff
@@ -97,15 +126,15 @@ def _end_location(cls, points):
 
     def save_to_db(self):
         db = get_section_db()
-        db.update({"_id": self._id}, 
+        db.update({"_id": self._id},
                       {"$set": {"distance" : self.distance, "mode" : self.mode, "confirmed_mode" : self.confirmed_mode}},
                        upsert=False, multi=False)
 
 
 class E_Mission_Trip(Trip):
 
-    def __init__(self, _id, user_id, trip_id, sections, start_time, end_time, trip_start_location, trip_end_location, alternatives, perturbed_trips, 
-                 mode_list, confirmed_mode_list):
+    def __init__(self, _id, user_id, trip_id, sections, start_time, end_time, trip_start_location, trip_end_location, alternatives, perturbed_trips,
+                 mode_list, confirmed_mode_list, recommended_alternative=None):
```

Comment:
@jimmydief, @sdsingh, why link from the trip to the recommendation? Why not from the recommendation to the original trip? That would allow flexibility to have multiple recommendations. An acceptable answer is "there's no particular reason, we just did something easy".

I just want to know so that we have a sense of what to change later.